backend/case/src/main/java/org/pucar/dristi/config/ServiceConstants.java

```diff
@@ -104,6 +106,7 @@ private ServiceConstants() {
 	public static final String STATE = "KL";
 	public static final String DISTRICT = "JL";
 	public static final String ESTABLISHMENT_CODE = "01";
+	public static final String COURTID="KLKM52";
```

Comment:
Establishment code is 52 for now and that's why the courtID is KLKM52.

and this should come from MDMS. why is this hard code?